Project_Writeup.txt

```diff
@@ -0,0 +1,36 @@
+Ilya Besancon
+Software Design Spring 2017
+MP3: Text Mining Reflection
+
+1) Project Overview:
+
+I used a twitter api to look at Chef Gordon Ramsay's last 200 tweets and used vader sentiment analysis.
+I was curious to look at if his tweets were generally positive or negative.
+
+2) Implementation
+
+From a system architectural perspective, this code is pretty simple. I used the twitter library,
+and used an api to gather the tweets I was interested in. I parsed them by using the .text function.
+This was actually my big design change, because initially I tried to manually parse the 'Text=' from the
+original list. Sentiment analysis is a pre-made package, and I just ran it through every individual tweet, and averaged
+that data to get my final result.
+
+3) Results
+
+Given the simple nature of this program, there isn't much to show for result.
+Here is the output of my program:
+Average Sentiment of 200 Chef Gordon Ramsay's Tweets:  0.181
```

Comment:
This value just by itself does not mean much to a reader. Giving some other values (like average sentiment value of other celebrities) to compare this value to will be helpful.